server.js

```diff
@@ -30,6 +32,8 @@ app.use(express.static(path.join(__dirname, '/client/build')));
 
 // Put all API endpoints under '/api'
 
+app.post('/api/findUser', userController.findUser);
```

Comment:
I wonder about the URL patterns here:  "/findUser". Camelcase seems a JS thing, not a typical URL pattern. We could change to find-user. But I also wonder if we have a plan to do more "user" operations. Do we want to keep on with the CRUD patterns? We don't need to. But maybe "/user/find" makes sense, or "/users/find" if we're gonna keep with the CRUD patterns.

Reply:
Good thinking. I'm not sure that we'll do too much more with Users, but it's better to have the naming conventions in place just in case.